configs/batch-llm/aws.yaml

```diff
@@ -1,84 +1,3 @@
 head_node_type:
-  name: head-node
-  instance_type: m5.2xlarge
-  resources:
-    cpu: 0
-worker_node_types:
-  - name: worker-g5-xlarge-nvidia-a10-1
-    instance_type: g5.xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-2xlarge-nvidia-a10-1
-    instance_type: g5.2xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-4xlarge-nvidia-a10-1
-    instance_type: g5.4xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-8xlarge-nvidia-a10-1
-    instance_type: g5.8xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-12xlarge-nvidia-a10-4
-    instance_type: g5.12xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-16xlarge-nvidia-a10-1
-    instance_type: g5.16xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 4
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-24xlarge-nvidia-a10-4
-    instance_type: g5.24xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-  - name: worker-g5-48xlarge-nvidia-a10-8
-    instance_type: g5.48xlarge
-    resources:
-      custom_resources:
-        "accelerator_type:A10G": 1
-    min_workers: 0
-    max_workers: 1
-    use_spot: true
-    fallback_to_ondemand: true
-aws:
-  TagSpecifications:
-    - ResourceType: instance
-      Tags:
-        - Key: as-feature-multi-zone
-          Value: "true"
+  name: head
```

Comment:
We will need a head node that has GPUs on it to run the python notebook due to the current need of eagerly importing vllm.